crates/ruff_linter/src/rules/flake8_pytest_style/rules/test_functions.rs

```diff
@@ -0,0 +1,82 @@
+use crate::checkers::ast::Checker;
+use crate::rules::flake8_pytest_style::rules::helpers::is_likely_pytest_test;
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, ViolationMetadata};
+use ruff_python_ast::{ParameterWithDefault, StmtFunctionDef};
+use ruff_text_size::Ranged;
+
+/// ## What it does
+/// Checks for parameters of test functions with default arguments.
+///
+/// ## Why is this bad?
+/// Such a parameter will always have the default value during the test
+/// regardless of whether a fixture with the same name is defined.
+///
+/// ## Example
+///
+/// ```python
+/// def test_foo(a=1): ...
+/// ```
+///
+/// Use instead:
+///
+/// ```python
+/// def test_foo(a): ...
+/// ```
+///
+/// ## Fix safety
+/// This rule's fix is marked as unsafe, as modifying a function signature can
+/// change the behavior of the code.
+///
+/// ## References
+/// - [Original Pytest issue](https://github.com/pytest-dev/pytest/issues/12693)
+#[derive(ViolationMetadata)]
+pub(crate) struct PytestParameterWithDefaultArgument {
+    parameter_name: String,
+}
+
+impl AlwaysFixableViolation for PytestParameterWithDefaultArgument {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!(
+            "Test function parameter `{}` has default argument",
+            self.parameter_name
+        )
+    }
+
+    fn fix_title(&self) -> String {
+        "Remove default argument".to_string()
+    }
+}
+
+/// PT028
+pub(crate) fn parameter_with_default_argument(
+    checker: &mut Checker,
+    function_def: &StmtFunctionDef,
+) {
+    if !is_likely_pytest_test(function_def, checker) {
+        return;
+    }
+
+    let parameters = function_def.parameters.as_ref();
+
+    for ParameterWithDefault {
+        parameter,
+        default,
+        range: pwd_range,
+    } in parameters.iter_non_variadic_params()
```

Comment:
This differs from the upstream rule that intentionally ignores `posonly` arguments

https://github.com/m-burst/flake8-pytest-style/blob/92c080312e36c13d2d7725d7bbd89d75d7eaae65/flake8_pytest_style/visitors/t_functions.py#L16-L23

Reply:
And PT028?